tasks/task_daily.json

```diff
@@ -0,0 +1,9 @@
+{
+"add_7mobile_action": true,
+"change_memory": false,
```

Comment:
hmm, can be turned on but not really know the difference since it's a sanity check.@@